components/expenses/PayExpenseModal.js

```diff
@@ -198,6 +199,27 @@ const getPaymentProcessorFee = (formik, expense, quoteQuery) => {
   }
 };
 
+const calculateAmounts = ({ formik, expense, quote, host, feesPayer }) => {
+  if (formik.values.forceManual) {
+    const totalAmount = { valueInCents: formik.values.totalAmountPaidInHostCurrency, currency: host.currency };
+    const paymentProcessorFee = {
+      valueInCents: formik.values.paymentProcessorFeeInHostCurrency,
+      currency: host.currency,
+    };
+    const VET = expense.currency !== host.currency && totalAmount.valueInCents / expense.amount;
```

Comment:
It's not clear to me what `VET` stands for, could we use a non-abbreviated name? i.e. `expenseToHostFxRate`

Reply:
Just noticed that VET concept is just brazilian. I'll replace by effective rate, because this is related to the rate between money sent and money received, including any fees.